docs/user/whats-new.asciidoc

```diff
@@ -1,12 +1,255 @@
+allowing you to summarize aggregated data.
+For information, check {kibana-ref}/lens.html#create-custom-tables[Create custom tables].
+
+[float]
+[[highlights-8.3-maps]]
+=== Enhancements to Maps
+
+[float]
+==== Language selector
+
+You can now select Arabic, German, English, Spanish,
+French, Hindi, Italian, Japanese, Portuguese, Russian, and Chinese for country
+and region labels. By default, new maps use the {kib}
+{kibana-ref}/i18n-settings-kb.html#general-i18n-settings-kb[locale setting] to determine
+the language for basemap labels.
+
+[role="screenshot"]
+image::images/highlights-maps-labels.gif[Menu for selecting a language for country and region labels]
+
+[float]
+==== Vector tiles and joins
+Are you using vector tiles? Good news. You are now able to make
+{kibana-ref}/terms-join.html[joins].
+
+[role="screenshot"]
+image::images/highlights-maps-vector-tiles.png[Joins for vector tiles in Maps]
+
+[float]
+[[highlights-8.3-security]]
+=== Improved Content Security Policy
+
+{kib} now has experimental support for a more restrictive
+https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP[Content Security Policy].
+Specifically, we removed the need for script-src ‘unsafe-eval’,
+which provides an additional level of protection against Cross-Site Scripting (“XSS”) attacks.
+
+To enable this feature, configure` csp.disableUnsafeEval: true` in your kibana.yml.
+
+[role="screenshot"]
+image::images/highlights-security.png[Setting for a more restrictive Content Security Policy]
+
+[float]
+[[highlights-8.3-ml]]
+=== Trained models test in Machine Learning
+
+You can now visualize machine learning trained model output.
+A new Test model action in *Machine Learning > Trained Models* opens up a
+flyout, where you can enter input text for testing. View the output
+using text highlighting or as JSON output.
+
+Trained models are available for the following tasks:
+language identification, named entity recognition (NER), text classification,
+zero-shot text classification, text embedding and fill task.
+
+[role="screenshot"]
+image::images/highlights-test-trained-model.png["Trained models test in Machine Learning", width=75%]
+
+
+[float]
+[[highlights-8.3-alerting]]
+=== Enhancements to Alerting
+
+[float]
+==== OAuth support in ServiceNow connectors
+
+The ServiceNow connectors now support open authentication (OAuth).
+For configuration details, refer to
+{kibana-ref}/servicenow-action-type.html[ServiceNow ITSM],
+{kibana-ref}/servicenow-sir-action-type.html[ServiceNow SecOps],
+and
+{kibana-ref}/servicenow-itom-action-type.html[ServiceNow ITOM connector].
+
+[float]
+==== Severity levels on cases
+
+You can now assign a severity level to each of your cases
+(critical, high, medium, or low). You can also view information about the
+average duration of your cases and delete comments in {kib}.
+
+[float]
+[[highlights-8.3-osquery]]
+=== Enhancements to Qsquery
```

Comment:
Small typo: Update `Qsquery` to `Osquery`

Reply:
fixed